server/src/optima/project.py

```diff
@@ -615,7 +615,15 @@ def uploadExcel():
         D.G.workbookname = D.G.projectname + '.xlsx'
         D.G.inputprograms = deepcopy(project.programs)
         D.G.inputpopulations = deepcopy(project.populations)
-        D = updatedata(D, input_programs = project.programs, savetofile = False)
+
+        # Is this the first time? if so then we have to run simulations
+        try:
+            D.S # the cool kids don't reRun when they already have simulations
+            shouldReRun = False
+        except AttributeError:
+            shouldReRun = True
```

Comment:
Thanks for pointing out the `_` as separator in Python style. I've read now about it now and indeed is the prevailing convention.

As I've said in http://stackoverflow.com/questions/8908760/should-i-use-camel-case-or-underscores-in-python
The "improve readability" has a little problem:
>*Function names should be lowercase, with words separated by underscores as necessary to improve readability.* 
>Which is absolutely contradictory with the fact that a camel cased group of words form an unambiguous solid symbol that looks like a single object matching the idea that a method name is one thing (as opposed to several objects, words, that have to be read and later interpreted as one single group of several things). That's even worst for _ if the method name contains a reserved word in it. 

But hey... millions are doing it, right?